core/src/main/java/org/apache/accumulo/core/iterators/system/LocalityGroupIterator.java

```diff
@@ -16,23 +16,28 @@
  */
 package org.apache.accumulo.core.iterators.system;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
We have a well-defined ordering of imports in Accumulo. I suggested that if these were re-ordered to adhered to that ordering, let's do that, just in a separate issue.

If the ordering was your just IDE moving stuff around, revert here is all we need.